jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibPluginIntegrationTest.java

```diff
@@ -157,9 +157,19 @@ public void testDockerContext() throws IOException, InterruptedException {
             .resolve("main")
             .resolve("resources")
             .resolve("newfile"));
-    BuildTask reexecutedJibDockerContextTask =
-        simpleTestProject.build("jibDockerContext").task(":jibDockerContext");
-    Assert.assertNotNull(reexecutedJibDockerContextTask);
-    Assert.assertEquals(TaskOutcome.SUCCESS, reexecutedJibDockerContextTask.getOutcome());
+    try {
+      BuildTask reexecutedJibDockerContextTask =
+          simpleTestProject.build("jibDockerContext").task(":jibDockerContext");
+      Assert.assertNotNull(reexecutedJibDockerContextTask);
+      Assert.assertEquals(TaskOutcome.SUCCESS, reexecutedJibDockerContextTask.getOutcome());
+
+    } catch (UnexpectedBuildFailure ex) {
```

Comment:
can you just catch the insecureRescursiveDeleteException directly?

Reply:
Since this is running in the `GradleRunner`, it doesn't actually expose the stacktrace from the task failure.